### ISSUE TITLE
[Accessibility] [Screen reader -Search Docs]: Screen reader is not giving the information about displayed suggestions. [Sev2][WCAG1.3.1]

### ISSUE THREAD

**Issue by Laxmidurga** [microsoft/tsdoc.org-website#30]
**Test Environment:**
OS Build: Windows 11
Version: 21H2 (OS Build 22000.318)
Browser: Edge
Version 96.0.1054.43 (Official build) (64-bit)
URL: https://tsdoc.org/
Screen reader: Narrator

**Repro Steps:**
1.	Open the above URL and login with valid credentials.
2.	TSDoc page will get displayed.
3.	Turn on screen reader and navigate to “Search Doc” field.
4.	Now, type any letter in the field (ex, micr), suggestions will appear. Observe whether the screen reader is narrating the information about displayed suggestions or not. 

**Actual:**
Screen reader is not giving the information about displayed suggestions after typing any text in the search docs field.
Screen reader is just stays silent after the suggestions displayed.

**Expected:**
When typing into the input field with auto-suggest, the screen reader should announce suggestions are available.
Screen reader should narrate like 5 suggestions are displayed, press down arrow key to navigate to the displayed suggestions.

**User Impact:**
Users who rely on screen readers need to be aware that autosuggestions will be provided based on what they type so they can get the benefit of this feature. The announcement is parallel to the visual list added to the page.

**WCAG Reference:**
[Understanding Success Criterion 1.3.1: Info and Relationships (w3.org)](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships)

https://user-images.githubusercontent.com/88436727/150468670-931bff93-d2c2-41ca-a0f8-e8e9009db228.mp4
